stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/ui/CloudAttachDialog.java

```diff
@@ -408,6 +408,7 @@ private void syncOrStash() {
     BasicAction.saveAll();
 
     if (syncResult == null) {
+      buildResult();
       checkSyncStashState();
```

Comment:
I think this method uses `log.error` instead of some more meaningful action. IDE error logger throws an exception to the top level and even suggest to disable plugin that caused it. It only makes sense to use it to log something really broken. Could we change the level to milder and decide what other warning we might show to the user if needed?